### PR TITLE
Implement an example about Logistic Regression in MesaPy

### PR DESCRIPTION
**Pull request by hi-T0day**
## Description
**Motivation:** 
Now, the two examples using Mesapy engine in MesaTEE ( Clef +1 ) are independent, one is about file access, the other one is about matrix_multiply. It is not enough. 

Therefore, this pull request implement an example including model training and prediction all in Python. The model is logistic regression. I expect this can provide another way for user to train the model and do the prediction. It is convenient for people who are familiar with Python to do secondary AI development in MesaTEE.

I finish all the algorithm by using Mesapy builtin library numpy. Actually, there are some differences between pypy micronumpy( MesaPy ) and numpy v1.17 (www.numpy.org). 

## Type of change (select applied and DELETE the others)

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
In my machine, I successfully compile and run the example: py_logistic_reg. 
The model paramaters and prediction result are the same to those in the Jupyter Notebook! 

However, I don't know how to do CI Check.

## Checklist (check ALL before submitting PR, even not applicable)

- [x] Fork the repo and create your branch from `master`.
- [x] If you've added code that should be tested, add tests.
- [x] If you've changed APIs, update the documentation.
- [ ] Ensure the tests pass (see CI results).
- [x] Make sure your code lints/format.

### CODE REVIEW

**Comment by mssun** [comment]
Thank you so much!

There are still some issues of our CI on the transition to the apache org. Don't worry, we will manually check the compilation and testing. In the meantime, we are working on the CI issue.

File: Makefile.deprecated [hunk 1]
```diff
@@ -75,7 +75,7 @@ SGX_MODULES := mesatee_services/kms mesatee_services/tdfs mesatee_services/tms \
 	mesatee_services/fns tests/functional_test
 SGX_LIBS :=
 UNIX_MODULES := integration_test private_join_and_compute quickstart \
-	image_resizing online_decrypt rsa_sign py_matrix_multiply py_file kmeans \
+	image_resizing online_decrypt rsa_sign py_matrix_multiply py_file py_logistic_reg kmeans \
```
[the remark this answers is not in the document]
Comment (hi-T0day) [reply Makefile.deprecated]: Okie. I use cmake to compile the code!

File: examples/py_logistic_reg/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[package]
+name = "py_logistic_reg"
+version = "0.1.0"
+authors = ["MesaTEE Authors <developers@mesatee.org>"]
+description = "File operations in Python with MesaTEE."
```
[the remark this answers is not in the document]
Comment (hi-T0day) [reply examples/py_logistic_reg/Cargo.toml]: Done!

**Comment by hi-T0day** [comment]
> There are still some issues of our CI on the transition to the apache org. Don't worry, we will manually check the compilation and testing. In the meantime, we are working on the CI issue.

Okie, I have read the mail-list. It seems that the GitHub Actions is helpful.

**Comment by mssun** [comment]
LGTM, thanks. We cannot merge PR at this time. Once the CI is setup, we will start to merge it.

**Comment by hi-T0day** [comment]
> LGTM, thanks. We cannot merge PR at this time. Once the CI is setup, we will start to merge it.

Okie :)

File: examples/py_logistic_reg/Cargo.toml (hunk 2 above)
[the remark this answers is not in the document]
Comment (litongxin1991) [reply examples/py_logistic_reg/Cargo.toml]: Thanks!

**Comment by mssun** [comment]
Merged, you can safely delete the branch. Thanks for your contribution.

**Pull request merged**